libs/expo/betterangels/src/lib/static/enumDisplayMapping.ts

```diff
@@ -21,6 +21,8 @@ export const enumDisplayGender: { [key in GenderEnum]: string } = {
   [GenderEnum.Female]: 'Female',
   [GenderEnum.Male]: 'Male',
   [GenderEnum.Other]: 'Other',
+  [GenderEnum.NonBinary]: 'Non-Binary',
+  [GenderEnum.PreferNotToSay]: 'Prefer not to say',
```

Comment:
why is this being updated here?

Reply:
just did migration and schema type generation.